Source/Urho3D/UI/UI.cpp

```diff
@@ -1308,6 +1309,7 @@ void UI::GetCursorPositionAndVisible(IntVector2& pos, bool& visible)
     if (cursor_ && cursor_->IsVisible())
     {
         pos = cursor_->GetPosition();
+        pos = ConvertUIToSystem(pos);

```

Comment:
Cursor is UI element and should works in the same coordinates as other UI elements. After your modification, there will be confusion, since the user will never be sure in which coordinates he is working

Reply:
The problem here is: If there is no cursor, i.e. system mouse cursor is used, the coordindate is "system", rather than "scaled UI position". In this case, there will inconsistent definition of the coordinate.

https://github.com/urho3d/Urho3D/blob/84913ce65b3aa0b8704a2e155d15e196dc572bdb/Source/Urho3D/UI/UI.cpp#L774
https://github.com/urho3d/Urho3D/blob/84913ce65b3aa0b8704a2e155d15e196dc572bdb/Source/Urho3D/UI/UI.cpp#L1318

Let's keep the discussion open.